tests/unit_tests/test_unit.py

```
@@ -797,20 +800,29 @@ def test_get_weights_from_url(monkeypatch):
         # file last modified
         # NOTE: Assuming test takes < 1 year to run
         curr_utc = datetime.datetime.now().astimezone(datetime.timezone.utc)
+        stale_time = (
+            datetime.datetime.now().timestamp()
```

Ruff (0.15.15)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)